Games/core/Meeting.js

```diff
@@ -756,6 +758,12 @@ module.exports = class Meeting {
 
     if (message.recipients.length == 0) return;
 
+    if (defaultRecipients) {
+      message.modified = true;
+  }
+
+  let anonymize = member.alive ? this.anonymous : false
```

Comment:
are you sure we want this fix? the dead will be able to see the meetings as non-anon

Reply:
you can also add a check for graveyeard participation : "all"